### PR TITLE
Added jamesbrine threat feed and parser

### PR DESCRIPTION
**Pull request by referefref**
# Commits

Added parser for jamesbrine threat feed
Added threat feed to feeds.yaml

# Commit Messages

Changes detailed in commit messages. Added parser instead of generic_csv due to date field not including timestamp, parser adds 00:00UTC.

# Description

Thanks

### CODE REVIEW

File: intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        'ip_address',
+        'activity' (unused),
+        'date'
+    ]
+
+    """
+
+    def parse_line(self, line, report):
+    if line.startswith('#') or len(line) == 0:
+        self.tempdata.append(line)
```
Comment (sebix) [inline intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py:30]: wrong indentation

File: intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2023 James Brine
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+""" IntelMQ parser for jamesbrine.com.au bruteforce IP feed """
+
+import dateutil.parser
+import posixpath
+from urllib.parse import urlparse
```
Comment (sebix) [inline intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py:9]: What are these imports for?

File: intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    [
+        'ip_address',
+        'activity' (unused),
+        'date'
+    ]
+
+    """
+
+    def parse_line(self, line, report):
+    if line.startswith('#') or len(line) == 0:
```
Comment (sebix) [inline intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py:29]: wong indentation

File: intelmq/etc/feeds.yaml [hunk 4]
```diff
@@ -1935,3 +1935,19 @@ providers:
               - true
               - false
               - true
+  JamesBrine:
+    Bruteforce Host Feed:
+      description: "jamesbrine.com.au provides an IPv4 blocklist covering brutefoce, exploit, and scanning activity from an international honeynet. The feed is available through https://jamesbrine.com.au"
```
Comment (sebix) [inline intelmq/etc/feeds.yaml]:
```suggestion
      description: "jamesbrine.com.au provides an IPv4 blocklist covering brutefoce, exploit, and scanning activity from an international honeynet"
```

File: intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py (hunk 1 above)
Comment (sebix) [inline intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py:30]: `self.tempdata` is never used. Did you intend to write your own `recover_line` method and using that one for `raw`?

**Review by sebix — CHANGES_REQUESTED** [review]
Thank you for your contribution.

Could you please add some tests? They would have shown some errors already

File: intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py (hunk 1 above)
Comment (referefref) [reply intelmq/bots/parsers/jamesbrine/parser_jamesbrine.py:30]: That was the intention. I've decided instead to adapt the csv endpoint instead such that a customer parser is not required and will be closing this PR.

**Comment by referefref** [comment]
CSV endpoint will be modified instead such that a custom parser is not required. PR will be closed and another opened when this has been completed and tested.

**Pull request closed without merging**